api/http/router.go

```diff
@@ -47,10 +47,11 @@ const (
 
 	APIURLManagement = "/api/management/v1/iot-manager"
 
-	APIURLSettings      = "/settings"
-	APIURLDevice        = "/devices/:id"
-	APIURLDeviceTwin    = "/devices/:id/twin"
-	APIURLDeviceModules = "/devices/:id/modules"
+	APIURLSettings = "/settings"
+
+	APIURLDevice                 = "/devices/:id"
+	APIURLDeviceState            = APIURLDevice + "/state"
+	APIURLDeviceStateIntegration = APIURLDevice + "/state/:integrationId"
```

Comment:
should integrationId be a path or query param?

Reply:
According to @alfrunes's docs, it's a path param.